packages/manager/apps/pci-private-registry/src/api/data/catalog.ts

```diff
@@ -0,0 +1,30 @@
+import { v6 } from '@ovh-ux/manager-core-api';
+
+export type TCatalog = {
+  addons: [
+    {
+      planCode: string;
+      blobs: {
+        technical: {
+          bandwidth: {
+            unlimited: boolean;
+          };
+        };
+      };
+      pricings: [
+        {
+          price: number;
+          tax: number;
+        },
+      ];
+    },
+  ];
+};
+
+export const getCatalog = async (ovhSubsidiary: string): Promise<TCatalog> => {
+  const { data } = await v6.get<TCatalog>(
+    `order/catalog/public/cloud?ovhSubsidiary=${ovhSubsidiary}`,
+  );
+
+  return data;
+};
```

Comment:
This api call with the type already exists within pci-common on packages/manager/modules/manager-pci-common/src/api/data/catalog.ts